pilot/pkg/model/telemetry.go

```diff
@@ -1028,73 +1028,13 @@ func stackdriverVMID(class networking.ListenerClass) string {
 	}
 }
 
-var metricToSDServerMetrics = map[string]string{
-	"REQUEST_COUNT":          "server/request_count",
-	"REQUEST_DURATION":       "server/response_latencies",
-	"REQUEST_SIZE":           "server/request_bytes",
-	"RESPONSE_SIZE":          "server/response_bytes",
-	"TCP_OPENED_CONNECTIONS": "server/connection_open_count",
-	"TCP_CLOSED_CONNECTIONS": "server/connection_close_count",
-	"TCP_SENT_BYTES":         "server/sent_bytes_count",
-	"TCP_RECEIVED_BYTES":     "server/received_bytes_count",
-	"GRPC_REQUEST_MESSAGES":  "",
-	"GRPC_RESPONSE_MESSAGES": "",
-}
-
-var metricToSDClientMetrics = map[string]string{
-	"REQUEST_COUNT":          "client/request_count",
-	"REQUEST_DURATION":       "client/response_latencies",
-	"REQUEST_SIZE":           "client/request_bytes",
-	"RESPONSE_SIZE":          "client/response_bytes",
-	"TCP_OPENED_CONNECTIONS": "client/connection_open_count",
-	"TCP_CLOSED_CONNECTIONS": "client/connection_close_count",
-	"TCP_SENT_BYTES":         "client/sent_bytes_count",
-	"TCP_RECEIVED_BYTES":     "client/received_bytes_count",
-	"GRPC_REQUEST_MESSAGES":  "",
-	"GRPC_RESPONSE_MESSAGES": "",
-}
-
 // used for CEL expressions in stackdriver serialization
 var jsonUnescaper = strings.NewReplacer(`\u003e`, `>`, `\u003c`, `<`, `\u0026`, `&`)
 
 func generateSDConfig(class networking.ListenerClass, telemetryConfig telemetryFilterConfig) *anypb.Any {
```

Comment:
why just metrics and not logs as well?

Reply:
Istio Stackdriver logs are not implemented using the OpenCensus SDK and are not removed in this PR.

Reply:
https://github.com/istio/proxy/pull/5546 says it will be removed ?

Reply:
It sounds good to me to remove Stackdriver logs from OSS Istio. To make the PR more focused, less error prone, and easier to review, removing Stackdriver logs from OSS Istio can be a separate PR.

Reply:
seems easier to remove them together since you can just delete tests instead of fixing them, but whatever you prefer